src/phoenix/server/api/types/Project.py

```diff
@@ -272,6 +273,17 @@ async def sessions(
         if after:
             cursor = Cursor.from_string(after)
             stmt = stmt.where(table.id < cursor.rowid)
+        if filter_condition:
+            span_filter = SpanFilter(condition=filter_condition)
```

Comment:
will this allow someone to just type raw text into the search field and it will search the messages? not seeing where that implicit search over messages is happening

Reply:
no. this is just a stopgap. see the unit test for an example